package.json

```diff
@@ -55,6 +56,7 @@
         "prettier": "^2.6.2",
         "rimraf": "^3.0.2",
         "style-loader": "^3.3.1",
+        "typescript": "4.7",
```

Comment:
For some reason this wasn't added as a dependency, 
Even though 4.7.3 it was in package-lock.json (not sure what happened or how the package-lock was generated).
this change ensures we have 4.7.4 defined in both files.